metagraph/scripts/gfa_merger.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python2
```

Comment:
What version of python should we use? I think that the header of 'exec_file_merging' function compiles only with python3 because of the object explicitation

Reply:
Python 2's deprecated, so let's not use anything that relies on it.